transform/mattermost-analytics/models/intermediate/web_app/int_mm_telemetry_prod__performance_events.sql

```diff
@@ -0,0 +1,10 @@
+{{
+    config({
+        "materialized": "incremental",
+        "incremental_strategy": "merge",
+        "cluster_by": ['timestamp'],
```

Comment:
[Snowflake documentation](https://docs.snowflake.com/en/user-guide/tables-clustering-keys#strategies-for-selecting-clustering-keys) has some suggestions on how to choose a clustering key. It is mentioned that `In general, if a column (or expression) has higher cardinality, then maintaining clustering on that column is more expensive.`. That's the reason clustering is usually done on a date rather than on a timestamp.

Reply:
Added a new field in staging, which extracts date from `timestamp` and using that as cluster_key, which I plan to also use in Looker.